src/ig/IGExporter.ts

```diff
@@ -983,6 +973,7 @@ export class IGExporter {
                   name ??
                   resourceJSON.id;
               }
+              newResource._sortKey = resourceJSON.id;
```

Comment:
It's possible that predefined resources are conformance resources. If so, they have a title and maybe a name, and I'd guess that's what the IG Publisher uses when creating the Table of Contents and Artifacts pages...  So, I wonder if this would be better as:
```ts
newResource._sortKey = resourceJSON.title ?? typeof resourceJSON.name === 'string' ? resourceJSON.name : resourceJSON.id;
```
That would more closely match what we do for profiles, extensions, etc.  What do you think?

Reply:
Actually, I just had a crazy thought.  All along, I've been assuming that the IG Publisher uses data from the resource (e.g., title, name, id) to determine what to display in the Table of Contents and Artifacts pages.  BUT... what if it is actually using `ImplementationGuide.definition.resource.name`??? I mean, now that I think about it, that might be the whole reason that you have to put a name in the `ImplementationGuide.definition.resource` in the first place!  If so, then (a) I feel dumb, (b) I'm sorry, and (c) that means you can just sort `ImplementationGuide.definition.resource` by `ImplementationGuide.definition.resource.name` and get rid of the `_sortKey`!  I think it's worth investigating.